flow-decomposition/src/main/java/com/powsybl/flow_decomposition/NetPositionComputer.java

```diff
@@ -33,6 +33,16 @@ static Map<Country, Double> computeNetPositions(Network network) {
             addLeavingFlow(netPositions, line, countrySide2);
         });
 
+        network.getTieLineStream().forEach(line -> {
+            Country countrySide1 = NetworkUtil.getTerminalCountry(line.getTerminal1());
+            Country countrySide2 = NetworkUtil.getTerminalCountry(line.getTerminal2());
+            if (countrySide1.equals(countrySide2)) {
+                return;
+            }
+            addLeavingFlow(netPositions, line, countrySide1);
+            addLeavingFlow(netPositions, line, countrySide2);
+        });
+
```

Comment:
I am not sure if they want this or not, I think they said it might cause issues in their computation (to check with Sebastien Murgey or Hugo Schindler)

Reply:
I don't see in the main code where tie lines are skipped.

Reply:
I approve this because only dangling lines are an issue for the flow decomposition